src/MiniCover/Reports/HtmlReport.cs

```diff
@@ -86,13 +87,25 @@ protected override void WriteDetailedReport(InstrumentationResult result, IDicti
                             style += BgColorBlue;
                         }
 
+                        var testMethods = kvFile.Value.Instructions
```

Comment:
I added a small icon to HTML report with list of tests.
It's very basic for now, just help on testing that functionality.